### PR TITLE
enabling ci cd for accelq ci plugin

### PR DESCRIPTION
**Pull request by accelq**
### Testing done

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
   <artifactId>accelq-ci-connect</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>${changelist}</version>
   <packaging>hpi</packaging>
 
   <properties>
     <revision>1.8</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/accelq-ci-connect-plugin</gitHubRepo>
```
Comment (NotMyFault) [inline pom.xml:20]:
```suggestion
```

File: pom.xml [hunk 2]
```diff
@@ -11,12 +11,12 @@
 
   <groupId>com.aq</groupId>
   <artifactId>accelq-ci-connect</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>${changelist}</version>
   <packaging>hpi</packaging>
 
   <properties>
     <revision>1.8</revision>
```
Comment (NotMyFault) [inline pom.xml:18]:
```suggestion
```

File: pom.xml (hunk 1 above)
Comment (accelq) [reply pom.xml:19]: done

File: pom.xml (hunk 2 above)
Comment (accelq) [reply pom.xml]: done

**Pull request merged**